keywords/SyncGateway.py

```diff
@@ -156,10 +159,17 @@ def start_sync_gateway(self, cluster_config, url, config):
         log_info("Starting sync_gateway on {} ...".format(target))
         ansible_runner = AnsibleRunner(cluster_config)
         config_path = os.path.abspath(config)
+
+        if is_cbs_ssl_enabled(cluster_config):
+            self.server_port = 18091
+            self.server_scheme = "https"
```

Comment:
The init of this class does not have the cluster_config, so initializing the default values in init and switching the values in start.